woocommerce/woocommerce
# Suppressing warnings for malformed metadata in product template downloads

### All Submissions:



### Changes proposed in this Pull Request:

This resolves a warning that is being triggered when a user creates a product via the digital download template. The warning results from corrupt meta-data on an image file that is downloaded (from [this source](https://woocommercecore.mystagingwebsite.com/wp-content/uploads/2017/12/album-1.jpg)) and then used within the product template as the product image. This doesn't cause any functional issue other than the warning when `WP_DEBUG` mode is set to `true`.

A couple options to resolve this:
1) Fix the EXIF data on the image itself, although this is sourced at `https://woocommercecore.mystagingwebsite.com/wp-content/uploads/2017/12/album-1.jpg`, and I'm unsure how this is edited or accessed.
2) We could consider this just too noisy as it would be triggered from any image with questionable metadata, with the solution being to suppress warnings in this instance.

I've taken option 2 for this PR. 

Closes #32137  .

### How to test the changes in this Pull Request:

1. Checkout branch
2. Ensure `WP_DEBUG` is set to true in `wp-config.php`
3. Go to the Home screen
4. Press Add my products on task list
5. And then press Start with a template
6. Select Digital product
7. Ensure no warning is displayed here or within the `debug.log`

### Other information:




### FOR PR REVIEWER ONLY:
